pynest/nest/lib/hl_api_types.py

```diff
@@ -460,9 +460,9 @@ def index(self, node_id):
 
         return index
 
-    def empty(self):
-        """Returns whether the NodeCollection is empty."""
-        return self.__len__() == 0
+    def __bool__(self):
+        """Converts the NodeCollection to a bool. False if it is empty, True otherwise."""
+        return self.__len__() > 0
```

Comment:
same as above, use
```suggestion
        return len(self) > 0
```
not only does it look better, it is actually faster (though I don't know why): 49 ns vs 62 ns on my machine

Reply:
On my computer there isn't any difference, both around 22.7 µs (my machine must be getting old). But I agree it looks better.